crates/api_models/src/payments.rs

```diff
@@ -4862,6 +4886,146 @@ pub struct PaymentsResponse {
     pub card_discovery: Option<enums::CardDiscovery>,
 }
 
+#[cfg(feature = "v2")]
+#[derive(Clone, Debug, serde::Serialize, ToSchema)]
+pub struct PaymentsListResponseItem {
+    /// Unique identifier for the payment. This ensures idempotency for multiple payments
+    /// that have been done by a single merchant.
+    #[schema(
+        min_length = 32,
+        max_length = 64,
+        example = "12345_pay_01926c58bc6e77c09e809964e72af8c8",
+        value_type = String,
+    )]
+    pub id: id_type::GlobalPaymentId,
```

Comment:
Can we remove this bit about the idempotency for the global payment IDs, since we are generating the IDs ourselves?